Add changelog (#571)
codex-team/codex.editor: README.md

Welcome to testing stage. Please, join a [public Telegram-chat](//t.me/codex_editor) where you always find a support.

### 2.2—2.7 changelog

- `New` *Sanitize API* — [Sanitize Config](https://github.com/codex-team/codex.editor/blob/master/docs/tools.md#automatic-sanitize) of `Block Tools` now automatically extends by tags of `Inline Tools` that is enabled by current Tool by `inlineToolbar` option. You don't need more to specify `a, b, mark, code` manually. This feature will be added to fields that supports inline markup.

> - `New` *Block Selection* — Ability to select Block by `CMD+A`, and the whole Editor by secondary `CMD+A`. After that, you can copy (`CMD+C`), remove(`Backspace`)or clear (`Enter`) selected Blocks.

Reviewer: ```suggestion
 - `New` *Block Selection* — Ability to select Block by `CMD+A`, and the whole Editor by double `CMD+A`. After that, you can copy (`CMD+C`), remove (`Backspace`) or clear (`Enter`) selected Blocks.
```

- `New` *[Styles API](https://github.com/codex-team/codex.editor/blob/master/types/api/styles.d.ts)* — Added `button` class for stylization of any buttons provided by Tools with one unified style.
- `New` *[Notifier API](https://github.com/codex-team/codex.editor/blob/master/docs/api.md#notifierapi)* — methods for showing user notifications: on success, errors, warnings, etc. 
- `New` *Block Tool* — [Table](http://github.com/codex-editor/table) constructor 💪 

> - `New` If one of Tools is unavailable on Editor initialization with saved data, its Blocks will be rendered with *Dummy Block*, describing that user can not edit content of this Block. Dummy Blocks can be moved, removed and saved as normal Blocks. So saved data won't be lost if one of Tools will failed 

Reviewer: ```suggestion
 - `New` If one of the Tools is unavailable on Editor initialization, its Blocks will be rendered with *Dummy Block*, describing that user can not edit content of this Block. Dummy Blocks can be moved, removed and saved as normal Blocks. So saved data won't be lost if one of the Tools is failed 
```

- `New` [Public TS-types](https://github.com/codex-team/codex.editor/tree/master/types) are presented.
- `Changes` *Tools API*  — options `irreplaceable` and `contentless` was removed.
- `Changes` *Tools API* — [Paste API](https://github.com/codex-team/codex.editor/blob/master/docs/tools.md#paste-handling): tags, patterns and mime-types now should be specified by Tool's `pasteConfig` static property. Custom Paste Event should be handled by `onPaste(event)` that should not be static from now.
- `Changes` *Tools API* — options `displayInToolbox ` and `toolboxIcon` was removed. Use [`toolbox`](https://github.com/codex-team/codex.editor/blob/master/docs/tools.md#internal-tool-settings) instead, that should return object with `icon` and `title` field, or `false` if Tool should not be placed at the Toolbox. Also, there are a way to override `toolbox {icon, title}` settings provided by Tool with you own settings at the Initial Config.
- `Improvements` — All Projects code now on TypeScript
- `Improvements` — NPM package size decreased from 1300kb to 422kb
- `Improvements` — Bundle size decreased from 438kb to 252kb

> - `Improvements` — `Inline Toolbar`: when you add a Link to the selected fragment, Editor will highlight this fragment event when Caret will be placed into the URL-input.

Reviewer: ```suggestion
 - `Improvements` — `Inline Toolbar`: when you add a Link to the selected fragment, Editor will highlight this fragment when Caret is set into the URL-input.
```

> - `Improvements` — Block Settings won't be showed near empty Blocks of `initialType`. You should click on them instead.

Reviewer: ```suggestion
 - `Improvements` — Block Settings won't be shown near empty Blocks of `initialType` by default. You should click on them instead.
```

> - `Improvements` — `onChange`-callback now will fired event on children attributes changing.

Reviewer: ```suggestion
 - `Improvements` — `onChange`-callback now is fired on children attributes change.
```
